File: Selenium/Activities/Activity11.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=service) as driver:

    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("Title: " + driver.title)
    checkbox = driver.find_element(By.CSS_SELECTOR, "input.willDisappear")
    print("Before: ",checkbox.is_selected())
    checkbox.click()
    print("After: ",checkbox.is_selected())
    driver.close()

File: Selenium/Activities/Activity11.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity11 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("Title: " + driver.getTitle());
        WebElement checkbox = driver.findElement(By.name("toggled"));
        System.out.println("Before: " + checkbox.isSelected());
        checkbox.click();
        System.out.println("After: " + checkbox.isSelected());
        driver.close();

    }

    }
